### DISCUSSION TITLE
Favorite LUT toggle

### DISCUSSION THREAD

**Discussion by KMOVIE8** [videovillage/community#87] [Ideas]
I have a huge list of LUTs on my system, and so it makes the workflow of toggling on a LUT for every single clip that I open quite a process- navigating though the menus and selecting the correct variation. I just wonder if there wouldn't be a better way to streamline this for a particular LUT that would be used for a session perhaps : 

say I am working in a particular color space today, and all the clips that I will be opening and playing back, it would be great to 1. open the clip 2. hit a keyboard shortcut toggle for turning the default (favorite) LUT on or off.

That's it!

**Comment by SteveOakley123** [comment]
Being able to have a couple of LUTs that you use all the time in a handy UI button + keystroke would be really really helpful. even better if the setting was persistent until changed.

 Reading the media file metadata which the app already does, and then picking the matching display LUT for "auto LUT" would be a big time saver.

**Comment by reichhen** [comment]
I 100% second this - I would like an option to set a default LUT for each particular file type/color space, as well as easily-chosen favorites.

**Comment by GAntiko** [comment]
This is definitely a must have. It's really time consuming to apply multiple times the same LUT. When you are watching many files is less effort to open davinci and apply a LUT to a timeline...

**Comment by GAntiko** [comment]
another good feature would be to apply a LUT to a playlist instead of a single video.

**Comment by Just-Jerr** [comment]
I just purchased Screen based on a number of online video professional comments who said it allowed easy LUT overlays and playlists which sounded ideal for quickly viewing my log footage at the end of a day of shooting.  I'm not finding any online manual for this software (if there is one, I'd love a link), but in my experimentation with creating playlists, selecting all clips and assigning my basic LUT for log footage the toolbar doesn't actually apply it to all of the selected clips (just the first one) and it has to be done individually a clip at a time which is a major time suck.  It seems like it would be a minimal programming task to be able to either apply a LUT to an entire playlist or have an option to default to having a chosen LUT applied automatically to every clip you're watching.  I feel  like that is generally what people in this thread have been asking for for awhile.  I'd love to hear a response as to whether this is a possibility or in development as I'm probably going to keep looking for a better solution at this point because while this looks slick and plays fine, it makes previewing card dumps extra tedious. So close to being a great time saver!

**Comment by SteveOakley123** [comment]
with Screen you have to copy setting from one clip, select others and paste. their workflow is a bit convoluted  but its still better than the alternatives. 

S

**Comment by gregcotten** [comment]
Working on something now to hopefully improve the ergonomics of doing this!

**Comment by SteveOakley123** [comment]
thank you !

ideally just make the initial clip view setting sticky between clips. generally tend to look at lots of the same clip type in a given session.

now if you read the file format metadata and auto set OEM LUT…. :)

S
